apps/server/src/modules/learnroom/imscc/imscc-file-builder.ts

```diff
@@ -0,0 +1,70 @@
+import { Readable } from 'stream';
+import { Builder } from 'xml2js';
+import AdmZip from 'adm-zip';
+import { randomUUID } from 'crypto';
+
+export class ImsccFileBuilder {
+	private readonly zipBuilder = new AdmZip();
+
+	private readonly xmlBuilder = new Builder({
+		rootName: 'manifest',
+	});
+
+	private readonly manifest = {
```

Comment:
The content of the manifest could be moved to an extra file to keep the builder free of the boilerplate code.

Reply:
I think we will refactor this in the future, when the builder grows.